stable/external-dns/Chart.yaml

```
@@ -4,7 +4,7 @@ description: |
   for Kubernetes Ingresses and Services
 name: external-dns
 version: 1.7.3
-appVersion: 0.5.12
+appVersion: 0.5.13
```

```suggestion
appVersion: 0.6.0
```

since it adds new feature